EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -165,25 +231,33 @@ public static string ComputePEPValuesForAllPSMsGeneric(List<PeptideSpectralMatch
 
                     lock (lockObject)
                     {
-                        ambiguousPeptidesRemovedCount += ambigousPeptidesRemovedinThread;
+                        ambiguousPeptidesResolved += ambigousPeptidesRemovedinThread;
                     }
                 });
+            return ambiguousPeptidesResolved;
+        }
 
-            var predictions = trainedModel.Transform(testData);
-
-            CalibratedBinaryClassificationMetrics metrics;
-            try
+        public static List<int>[] Get_PSM_Group_Indices(int count, int numGroups)
```

Comment:
"numPSMs" would be a more descriptive variable name than "count"

Reply:
changed it to psmsCount, which is what is used in the function call "List<int>[] psmGroupIndices = Get_PSM_Group_Indices(psms.Count, numGroups);"